openwisp_monitoring/device/settings.py

```diff
@@ -58,3 +58,9 @@ def get_health_status_labels():
 MAC_VENDOR_DETECTION = getattr(
     settings, 'OPENWISP_MONITORING_MAC_VENDOR_DETECTION', True
 )
+
+DEVICE_RESOURCES_THRESHOLDS = getattr(
+    settings,
+    'OPENWISP_MONITORING_DEVICE_RESOURCES_THRESHOLDS',
+    {'memory': 95, 'load': 90, 'disk_usage': 80},
```

Comment:
Let's use these as default thresholds: `free_memory`, `load_1` and `free_space` keys.

Reply:
A threshold value needs to be a percentage here, if I am not wrong. The reason being `free_memory` and `free_space` will vary from device to device we can't set a default threshold as per one device, right? I can use `load_1` as a threshold because it's value will lie between 0 and 1 always.

Reply:
are you sure it's comprised between 0 and 1 only? I knew it can be more than 1, especially on multi core devices. A load of 2 with 2 cores is the same of 1 with 1 core.
But couldn't also happen to have a load of 1.1 with 1 core @PabloCastellano? If I'm not wrong this can happen and indicates an overload.

Reply:
I never observed it as my device (VM) is single core :sweat_smile:.
Should I use ```load = load_5/(no.of cores)```, that should be consistent, I guess.